elements/lisk-codec/test/add_schema.spec.ts

```diff
@@ -12,6 +12,47 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { codec } from '../src/codec';
+
+const testSchema = {
```

Comment:
can we use `test/fixtures`? as we reuse lot of this schema.

Reply:
not sure what you mean

Reply:
Need to rewrite this test anyway so maybe clarify so I can use the suggestion when doing so.

Reply:
you are using the test schema in `add_schema.spec`, `encode.spec.ts ` and maybe in future other places too. So if you add these common usage to `test/fixtures` folder and refer it from there, then we can avoid the duplication, thats the only intention.

Reply:
Oh got it; I think we should use the same schemas from protocol specs in general so that will be as you suggest. Then we need to decide what to do with extra schemas we come up with (currently protocol-specs are linked to this folder)